CLAC

```diff
@@ -0,0 +1,157 @@
+                       DEDIS / EPFL / Lausanne / CH
+  Software Grant and Corporate Contributor License Agreement ("Agreement")
+                      http://www.apache.org/licenses/
+
+   Thank you for your interest in software from DEDIS (Lab).
+   In order to clarify the intellectual property license
+   granted with Contributions from any person or entity, the Lab
+   must have a Contributor License Agreement (CLA) on file that has been
+   signed by each Contributor, indicating agreement to the license terms
+   below. This license is for your protection as a Contributor as well
+   as the protection of the Lab and its users; it does not change
+   your rights to use your own Contributions for any other purpose.
+
+   This version of the Agreement allows an entity (the "Corporation") to
+   submit Contributions to the Lab, to authorize Contributions 
+   submitted by its designated employees to the Lab, and to grant 
+   copyright and patent licenses thereto.
```

Comment:
The code is under the AGPL3: 
- If you change the code and use and/or distribute it, even in the cloud, you have to publish the changes.
Regarding our repo:
- If you want your code to be included in our repo, you need to sign off the `CLA[IC]`.
Commercial use:
- If you want to change the code and not publish the changes, or if you want us to work on the code and keep the result for yourself, you get a special, paying license.